lib/badge-data.js

```diff
@@ -20,7 +20,11 @@ function isDataUri(s) {
 function prependPrefix(s, prefix) {
```

Comment:
`s` is expected to be a string, so if we don't get one, it seems we should either coerce to a string or throw an exception.